example_test.go

```diff
@@ -12,6 +13,29 @@ import (
 	"github.com/creachadair/tomledit/parser"
 )
 
+func Example_replaceStrValues() {
+	data := `
+	[ A.B ]
+	C = "Vc"
+	`
+	doc, err := tomledit.Parse(strings.NewReader(data))
+	if err != nil {
+		log.Fatalf("Parse: %v", err)
+	}
+
+	target := doc.First("A", "B", "C")
+	target.Value = parser.MustValue(`"Vc2"`).WithComment("replaced!")
+
+	var buf bytes.Buffer
+	if err := tomledit.Format(&buf, doc); err != nil {
+		log.Fatalf("Format: %v", err)
+	}
+	fmt.Println(buf.String())
```

Comment:
It's not necessary to buffer the output, this could equivalently be:
```suggestion
	tomledit.Format(os.Stdout, doc)
```

(or do like line 99 and check the error, if you want)